test/parseNuspec-no-dependencies.spec.ts

```diff
@@ -114,9 +114,4 @@ describe('parseNuSpec', () => {
       version: '472'},'dependencyName')).rejects.toThrow();
   });
 
-  it('should throw an error when the nuspec contains non-array dependencies tag', async () => {
```

Comment:
can we delete this fixture now too? is it used anywhere else?

Reply:
you are right! I will delete it